lib/3_add_up_numbers.js

```diff
@@ -4,8 +4,15 @@
 // 
 // Run `bin/check` from the command line to execute the automated tests.
 
+//module.exports = {
+//  add_up_numbers: function(arr){
+//var sum = arr.reduce((pv, cv) => pv+cv, 0);
+//  return sum;
+//  }
+//}
+
 module.exports = {
   add_up_numbers: function(arr){
-    // Your code goes here.
-  }
-}
\ No newline at end of file
+    var sum = arr.reduce(function(pv, cv) { return pv + cv; }, 0);
```

Comment:
Can you tell me more about your inclusion of `, 0` after the block that executes in this anonymous function? Is that an argument that is passed to `#reduce`?